shell/browser/api/electron_api_base_window.cc

```diff
@@ -1101,8 +1101,10 @@ void BaseWindow::SetAppDetails(const gin_helper::Dictionary& options) {
   options.Get("relaunchCommand", &relaunch_command);
   options.Get("relaunchDisplayName", &relaunch_display_name);
 
-  ui::win::SetAppDetailsForWindow(app_id, app_icon_path, app_icon_index,
-                                  relaunch_command, relaunch_display_name,
+  ui::win::SetAppDetailsForWindow(base::UTF16ToWide(app_id), app_icon_path,
+                                  app_icon_index,
+                                  base::UTF16ToWide(relaunch_command),
+                                  base::UTF16ToWide(relaunch_display_name),
```

Comment:
can these be `std::wstring`s that get converted during `options.Get` directly, rather than converting from JS string -> u16string -> wstring?

Reply:
Yes. 0f3620099a3de57956370e3c463d02bd0b2dc824